Frameworks/UIKit/UILabel.mm

```diff
@@ -44,61 +44,70 @@ @implementation UILabel {
     UILineBreakMode _lineBreakMode;
     BOOL _adjustFontSize;
     float _minimumFontSize;
-    float _originalFontSize;
     int _numberOfLines;
     BOOL _isDisabled;
     BOOL _isHighlighted;
     UIBaselineAdjustment _baselineAdjustment;
     StrongId<WXCTextBlock> _textBlock;
 }
 
-- (void)adjustFontSizeToFit {
+- (void)_adjustFontSizeToFit {
     if (_numberOfLines == 0) {
```

Comment:
I got rid of it and adding additional test case for multi-line adjustWidthToFit. 

---
In reply to: [97388485](https://github.com/Microsoft/WinObjC/pull/1768#discussion_r97388485) [](ancestors = 97388485)